client/a8c-for-agencies/components/items-dashboard/item-preview-pane/index.tsx

```diff
@@ -96,8 +97,9 @@ export default function ItemPreviewPane( {
 				<SectionNav
 					className={ clsx( 'preview-pane__navigation', { 'is-hidden': shouldHideNav } ) }
 					selectedText={ selectedFeature.tab.label }
+					enforceTabsView={ enforceTabsView }
 				>
-					{ navItems && navItems.length > 0 ? (
+					{ ! shouldHideNav && navItems && navItems.length > 0 ? (
 						<NavTabs hasHorizontalScroll>{ navItems }</NavTabs>
```

Comment:
Should this be using `ResponsiveToolbarGroup`?

Reply:
Probably not, it's more tag selection than tab selection. :thinking:

Reply:
It would be better if we could use the TabList component from the Gutenberg. But it might be a big change.

Reply:
> Should this be using ResponsiveToolbarGroup?

We cannot because the overflow buttons will be combined into a "More" dropdown.